scripts/feedme.coffee

```diff
@@ -0,0 +1,118 @@
+# Description:
+#   Shows what"s edible.
+#
+# Notes:
+#   This includes
+#    * fetching from fabrik
+#    * fetching from ernis
+#
+# Author: cholter
+
+urllib = require "urllib-sync"
+jsdom = require "jsdom"
+require "datejs"
+
+DAY = 1000 * 60 * 60  * 24
+
+getDateDiff = (date1, date2, interval) ->
+  return Math.round((date1.getTime() - date2.getTime()) / interval)
+
+fetch = (target) ->
+  return urllib.request(target).data
+
+
+class Fabrik
+  target : "http://www.diefabrik.co.at/mittagsmenue/index.html"
+  holidayMagic : "urlaub"
+  errors:
+    closed : "fabrik is closed"
```

Comment:
Formatting of lines 28 through 32 is inconsistent with formatting of line 25 and 26.